### PR TITLE
Support server side DESCRIBE command in 4.0

### LINKED ISSUE #199: Support server side DESCRIBE command in 4.0
In Cassandra 4.0 a server side DESCRIBE command was introduced. Today ecaudit logs a warning about unrecognized command for this but it still audit logged but we should add proper support for this command.

### PR DESCRIPTION
**Pull request by tommystendahl**
Closes #199

### CODE REVIEW

**Review by jwaeab — APPROVED** [review]

File: ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/factory/AuditEntryBuilderFactory.java [hunk 1]
```diff
@@ -184,6 +185,10 @@ private Builder createEntryBuilder(Raw parsedStatement)
         {
             return createBatchEntryBuilder();
         }
+        if (parsedStatement instanceof DescribeStatement)
+        {
```
Comment (masokol) [inline ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/factory/AuditEntryBuilderFactory.java:189]: Strange formatting

File: ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/factory/AuditEntryBuilderFactory.java [hunk 2]
```diff
@@ -224,6 +229,10 @@ public Builder createEntryBuilder(CQLStatement statement)
         {
             return createBatchEntryBuilder();
         }
+        if (statement instanceof DescribeStatement)
+        {
```
Comment (masokol) [inline ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/factory/AuditEntryBuilderFactory.java:233]: Strange formatting

File: ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/factory/AuditEntryBuilderFactory.java (hunk 1 above)
Comment (tommystendahl) [reply ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/factory/AuditEntryBuilderFactory.java:189]: Fixed

File: ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/factory/AuditEntryBuilderFactory.java (hunk 2 above)
Comment (tommystendahl) [reply ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/factory/AuditEntryBuilderFactory.java:233]: Fixed

**Review by masokol — COMMENTED** [review]
LGTM

**Pull request merged**